sanic/response.py

```diff
@@ -195,8 +195,14 @@ def output(
         timeout_header = b''
         if keep_alive and keep_alive_timeout is not None:
             timeout_header = b'Keep-Alive: %d\r\n' % keep_alive_timeout
-        self.headers['Content-Length'] = self.headers.get(
-            'Content-Length', len(self.body))
+
+        body = b''
+        content_length = 0
+        if self.status is not 204 and self.status != 304:
```

Comment:
Whoops, I approved this already but a minor nitpick here: `if self.status is not 204 and self.status != 304` can we use the same comparison operator? Or, another idea is to put these into a constant list at the top: `EMPTY_RESPONSE_CODES = [204, 304]` and then do `if self.status not in EMPTY_RESPONSE_CODES`. This way we can add more response codes easily if needed.

Reply:
Sure I will fix them, I’ll do the empty list thing :)